ray-operator/main.go

```diff
@@ -218,9 +222,9 @@ func main() {
 	ctx := ctrl.SetupSignalHandler()
 	exitOnError(ray.NewReconciler(ctx, mgr, rayClusterOptions).SetupWithManager(mgr, config.ReconcileConcurrency),
 		"unable to create controller", "controller", "RayCluster")
-	exitOnError(ray.NewRayServiceReconciler(ctx, mgr, utils.GetRayDashboardClient, utils.GetRayHttpProxyClient).SetupWithManager(mgr),
```

Comment:
I don't think it's necessary at the moment since we're only using a single field there. But we should revisit this in the future and when config API graduates to beta. What do you think?

Reply:
I recently worked on an open-source event for newbies. This would be a good first issue for the event. I will open an issue later.